hack/install-controller-gen.sh

```diff
@@ -13,7 +13,7 @@ then
 fi
 
 
-CMDPATH="$GOPATH/pkg/mod/sigs.k8s.io/controller-tools@v0.4.0/cmd/controller-gen"
+CMDPATH="$GOPATH/pkg/mod/sigs.k8s.io/controller-tools@$1/cmd/controller-gen"
```

Comment:
Can we maybe default this here (like `${1:-v0.4.1}`) in case somebody is running this script directly rather than via the makefile?

Reply:
@zaneb Yep, done, rebased PR